pycxdgui/startGUI.py

```diff
@@ -13,7 +15,7 @@
 SDIR = "../storage"
 sxsdesc = "B002.sxs"
 
-configfile = "saxsgui-config.yml"
+configfile = pkg_resources.resource_filename("pycxdgui", "saxsgui-config.yml")
```

Comment:
This way the installed program will recognize where to look for the config file.

Reply:
yeah we might want to search through a list of paths in the future. but for now this is good. thanks!